src/NServiceBus.Core.Tests/.editorconfig

```diff
@@ -7,3 +7,6 @@ dotnet_diagnostic.IDE0074.severity = none
 
 # Justification: test project
 dotnet_diagnostic.CA2007.severity = none
+
+# IDE0028: Simplify collection initialization
```

Comment:
**Nitpick**: Can you please update this comment to have the same format as the others, meaning: "Justification:..."